Compile error, multiple default-values specified

Compiling .pio/build/nodemcuv2/libf5a/MCP23017_ID6001/MCP23017.cpp.o
.pio/libdeps/nodemcuv2/MCP23017_ID6001/src/MCP23017.cpp:3:56: error: default argument given for parameter 2 of 'MCP23017::MCP23017(uint8_t, TwoWire&)' [-fpermissive]
 MCP23017::MCP23017(uint8_t address, TwoWire& bus = Wire) {
                                                        ^
In file included from .pio/libdeps/nodemcuv2/MCP23017_ID6001/src/MCP23017.cpp:1:0:
.pio/libdeps/nodemcuv2/MCP23017_ID6001/src/MCP23017.h:67:2: note: previous specification in 'MCP23017::MCP23017(uint8_t, TwoWire&)' here
  MCP23017(uint8_t address, TwoWire& bus = Wire);

Default parameter has been specified both here https://github.com/blemasle/arduino-mcp23017/blob/master/src/MCP23017.cpp#L3 and here https://github.com/blemasle/arduino-mcp23017/blob/master/src/MCP23017.h#L67
It should only be set in header-file!

--- a/src/MCP23017.cpp
+++ b/src/MCP23017.cpp
@@ -1,6 +1,6 @@
 #include "MCP23017.h"
 
-MCP23017::MCP23017(uint8_t address, TwoWire& bus = Wire) {
+MCP23017::MCP23017(uint8_t address, TwoWire& bus) {
 	_deviceAddr = address;
 	_bus = &bus;
 }

